Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.h

```diff
@@ -189,6 +198,8 @@ class RemoteRenderingBackendProxy
     HashMap<MarkSurfacesAsVolatileRequestIdentifier, CompletionHandler<void(bool)>> m_markAsVolatileRequests;
     SerialFunctionDispatcher& m_dispatcher;
 
+    HashSet<Client*> m_clients;
```

Comment:
This breaks our smart pointe rules.

Can we do this without storing RemoteLayerWithRemoteRenderingBackingStoreCollections as clients here? There's only ever one RemoteLayerBackingStoreCollection per context, so it would be cleaner to find the RemoteLayerBackingStoreCollection by going via the WebPageProxy I think.